the-collab-lab/tcl-50-smart-shopping-list
# Installed and setup tailwindcss

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description
I installed and setup tailwindcss 


## Related Issue
Closes #33  


## Acceptance Criteria
UI-related tasks:
- [x] Install and setup TailwindCss


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before



### After



## Testing Steps / QA Criteria
1. From your terminal, do `git checkout main`
2. Pull down this branch with `git pull origin md-setup-install-tailwind`
3. Checkout the branch with `git checkout md-setup-install-tailwind`
4.  To test, add a style to any element. You can start with a background color: `bg-sky-500` to a div or delete button. Check this [tailwind doc](https://tailwindcss.com/docs/text-color) for more.
